shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/execute/engine/driver/jdbc/JDBCExecutorCallback.java

```diff
@@ -66,6 +66,9 @@
                 result.add(executeResult);
             }
         }
+        if (dataMap.containsKey(ExecuteProcessConstants.EXECUTE_ID.name())) {
```

Comment:
Why do not we put this into `DriverJDBCExecutor` so that it will correspond to `initialize()`?

Reply:
Since

- `finishReport(dataMap, executionUnit)` is in `JDBCExecutorCallback.java `, and it has `if (dataMap.containsKey(ExecuteProcessConstants.EXECUTE_ID.name()))`, make `finish` correspond to it
- it has less places to add this piece of code if we add it in `JDBCExecutorCallback` and `RawSQLExecutorCallback` than in `DriverJDBCExecutor`, `ProxyJDBCExecutor`, `RawExecutor` and `CalciteRowExecutor`

Reply:
So we can not move these coding into `ExecuteProcessEngine` as `finish()` like other functions in this class?
Plus, we have called `ExecuteProcessEngine.initialize` in all the places. I mean `JDBCExecutorCallback` is for  SQL process and report each child status, on the other hand, `DriverJDBCExecutor` is the main entrance for `initialize` and `finish`.